packages/office-addin-dev-settings/src/sideload.ts

```diff
@@ -380,3 +377,16 @@ export async function sideloadAddIn(
     throw err;
   }
 }
+
+async function writeZipFile(zip: jszip, pathToWrite: string): Promise<string> {
+  // Write the file
+  await new Promise((resolve, reject) => {
+    zip
+      .generateNodeStream({ type: "nodebuffer", streamFiles: true })
+      .pipe(fs.createWriteStream(pathToWrite))
+      .on("error", reject)
+      .on("finish", resolve);
+  });
+
+  return pathToWrite;
```

Comment:
Why do we need to return the same value that is passed in?  Seems like it's extra.  If we return anything I would think it would be a boolean indicating success of the file write . . . but we weren't doing that before so it would just be a bonus if we did it now.

Reply:
I just removed the function. It was not needed any longer after some of the previous refactorings